chain/epoch-manager/Cargo.toml

```diff
@@ -18,6 +18,7 @@ rand_hc.workspace = true
 serde_json.workspace = true
 smart-default.workspace = true
 tracing.workspace = true
+itertools.workspace = true
```

Comment:
Yeah I usually use collect_vec when printing debug logs. How would you feel about leaving that in?

Reply:
Generally speaking, I don't really like adding dependencies that aren't used in committed code. It's the kind of thing I personally might create a PR, just to remove a dead dependency. So at least a comment why it's in would be useful to prevent that sort of engineers canceling out each others "improvements".

But that said, itertools wouldn't be the worst to have pulled in for no reason, so I won't stop you from adding it if you hate writing `collect::<Vec<_>>` that much :D

Reply:
Ok, thanks, I'll leave that in and add a comment.